index.js

```diff
@@ -0,0 +1,123 @@
+module.exports = {
+    "plugins": [
+        "import"
+    ],
+    "extends": "eslint:recommended",
+    "rules": {
+        "no-unsafe-negation": "error",
+        "curly": [
+            "error",
+            "all"
+        ],
+        "default-case": "error",
+        "dot-location": [
+            "error",
+            "property"
+        ],
+        "dot-notation": "error",
+        "eqeqeq": "error",
+        "no-caller": "error",
+        "no-else-return": "error",
+        "no-empty-function": "error",
+        "no-eq-null": "error",
+        "no-extra-bind": "error",
+        "no-floating-decimal": "error",
+        "no-global-assign": "error",
+        "no-implicit-coercion": "error",
+        "no-implicit-globals": "error",
+        "no-implied-eval": "error",
+        "no-iterator": "error",
+        "no-lone-blocks": "error",
+        "no-multi-spaces": "error",
+        "no-new-func": "error",
+        "no-new-wrappers": "error",
+        "no-octal-escape": "error",
+        "no-param-reassign": [
+            "error",
+            {
+                "props": false
+            }
+        ],
+        "no-proto": "error",
+        "no-return-assign": [
+            "error",
+            "always"
+        ],
+        "no-self-assign": [
+            "error",
+            {
+                "props": false
+            }
+        ],
+        "no-self-compare": "error",
+        "no-sequences": "error",
+        "no-throw-literal": "error",
+        "no-useless-call": "error",
+        "no-useless-concat": "error",
+        "no-useless-escape": "error",
+        "no-with": "error",
+        "radix": [
+            "error",
+            "as-needed"
+        ],
+        "strict": [
+            "error",
+            "safe"
+        ],
+        "no-shadow-restricted-names": "error",
+        "no-use-before-define": "error",
+        "array-bracket-spacing": [
+            "error",
+            "never"
+        ],
+        "block-spacing": "error",
+        "brace-style": "error",
+        "comma-dangle": [
+            "error",
+            "never"
+        ],
```

Comment:
I recently came across AirBnB style guide and they require trailing commas as it makes the diff smaller.
what do you guys think about it?
http://eslint.org/docs/rules/comma-dangle